IBAnimatable/AnimatableScrollView.swift

```diff
@@ -0,0 +1,153 @@
+        didSet {
+            configFillColor()
+        }
+    }
+    
+    @IBInspectable public var predefinedColor: String? {
+        didSet {
+            configFillColor()
+        }
+    }
+    
+    @IBInspectable public var opacity: CGFloat = CGFloat.nan {
+        didSet {
+            configOpacity()
+        }
+    }
+    
+    // MARK: - BorderDesignable
+    @IBInspectable public var borderColor: UIColor? {
+        didSet {
+            configBorder()
+        }
+    }
+    
+    @IBInspectable public var borderWidth: CGFloat = CGFloat.nan {
+        didSet {
+            configBorder()
+        }
+    }
+    
+    @IBInspectable public var borderSide: String? {
+        didSet {
+            configBorder()
+        }
+    }
+    
+    // MARK: - RotationDesignable
+    @IBInspectable public var rotate: CGFloat = CGFloat.nan {
+        didSet {
+            configRotate()
+        }
+    }
+    
+    // MARK: - ShadowDesignable
+    @IBInspectable public var shadowColor: UIColor? {
+        didSet {
+            configShadowColor()
+        }
+    }
+    
+    @IBInspectable public var shadowRadius: CGFloat = CGFloat.nan {
+        didSet {
+            configShadowRadius()
+        }
+    }
+    
+    @IBInspectable public var shadowOpacity: CGFloat = CGFloat.nan {
+        didSet {
+            configShadowOpacity()
+        }
+    }
+    
+    @IBInspectable public var shadowOffset: CGPoint = CGPoint(x: CGFloat.nan, y: CGFloat.nan) {
+        didSet {
+            configShadowOffset()
+        }
+    }
+    
+    // MARK: - BlurDesignable
+    @IBInspectable public var blurEffectStyle: String?
+    @IBInspectable public var blurOpacity: CGFloat = CGFloat.nan
+    
+    // MARK: - TintDesignable
+    @IBInspectable public var tintOpacity: CGFloat = CGFloat.nan
+    @IBInspectable public var shadeOpacity: CGFloat = CGFloat.nan
+    @IBInspectable public var toneColor: UIColor?
+    @IBInspectable public var toneOpacity: CGFloat = CGFloat.nan
+    
+    // MARK: - GradientDesignable
+    @IBInspectable public var startColor: UIColor?
+    @IBInspectable public var endColor: UIColor?
+    @IBInspectable public var predefinedGradient: String?
+    @IBInspectable public var startPoint: String?
+    
+    // MARK: - MaskDesignable
+    @IBInspectable public var maskType: String? {
+        didSet {
+            configMask()
+            configBorder()
+        }
+    }
+    
+    // MARK: - Animatable
+    @IBInspectable public var animationType: String?
+    @IBInspectable public var autoRun: Bool = true
+    @IBInspectable public var duration: Double = Double.nan
+    @IBInspectable public var delay: Double = Double.nan
+    @IBInspectable public var damping: CGFloat = CGFloat.nan
+    @IBInspectable public var velocity: CGFloat = CGFloat.nan
+    @IBInspectable public var force: CGFloat = CGFloat.nan
+    @IBInspectable public var repeatCount: Float = Float.nan
+//    @IBInspectable public var x: CGFloat = CGFloat.nan
```

Comment:
This is a change from Ahtau, not intended for IBAnimatable. I have written other extensions for UIView that make use of x and y values without the Animatable protocol in place.